app/controllers/permissions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PermissionsController < ApplicationController
+	include PermissionsHelper
```

Comment:
Can you change the controller name?

Reply:
replaced this Permissionscontroller with the existing one, which is the manage_users controller.